pom.xml

```diff
@@ -22,9 +21,13 @@
     </properties>
 
     <dependencies>
+
+        <!-- https://mvnrepository.com/artifact/junit/junit -->
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-thymeleaf</artifactId>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
we haven't written any tests yet so I would suggest you remove this dependency for now, in addition Junit released a new Junit5 version, we can give that a try when we start writing tests.